src/components/Content.tsx

```diff
@@ -0,0 +1,136 @@
+/** @jsxImportSource @emotion/react */
+import { css } from '@emotion/react';
+import PropTypes from 'prop-types';
+import userData from '../db/user.json';
+import { useEffect, useRef } from 'react';
+
+// 메시지 타입
+interface messageType {
+  id: number;
+  userId: number;
+  text: string;
+  time: string;
+}
```

Comment:
`messageType`이 다른 파일에서도 사용 중인데, 각각의 파일에서 타입을 정의하고 있어 중복을 줄이면 좋을 것 같아요!
또한 유지보수 측면에서 추후에 수정이 필요할 때 여러 곳에서 동일한 작업을 반복해야 하는 문제가 발생할 수 있습니다!

Reply:
타입을 정의하는 별도의 파일을 만들고 거기에서 각각을 export 해서 사용하면 좋을 것 같아요~
```suggestion
// src/interfaces/message.ts
export interface MessageType {
  id: number;
  userId: number;
  text: string;
  time: string;
}
```
```javascript
import { MessageType } from 'src/interfaces/message
```